openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -37,12 +37,16 @@
 - control_key: IA-2 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
```

Comment:
```suggestion
  implementation_status: not applicable
```

Reply:
@kevindoingbizas Since this is completely deferred to IdP, what do you think of tagging this as ``not applicable``?

Reply:
Agree..

Reply:
Ok. Merging this suggestion.